### PR TITLE
Template 3 - Mobile version skeleton

### PR DESCRIPTION
**Pull request by TEMESGEN-G25**
In this project I:
- Added one HTML file & one CSS file.
- A mobile skeleton from one of MIcroverse's figma templates.
- Added icons from font awesome.

### CODE REVIEW

**Review by aliabbani — COMMENTED** [review]
Hi @NatiG25 ✋  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

The linter in your repository should always be green, Please solve the Stylelint errors and submit again 😊 .

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Lordkaito — COMMENTED** [review]
Hi @NatiG25 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not passed linters. Please make sure to correct this before asking for a new code review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mike2611 — CHANGES_REQUESTED** [review]
Hi @NatiG25 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has not passed linters. Please make sure to correct this before asking for a new code review.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header class="header">
+      <p class="logo">Welcome</p>
+      <img src="image/Union.svg" alt="Menu-button" class="hamburger-menu">
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] you did well but I think you can still improve your code by using semantic elements. I suggest you use the `nav` to wrap the `image` tag at line 14. Please, make sure that your name is also wrapped in an anchor `a` element. By this, your name will be clickable and navigates to wherever you may want to take the users.
   Example
   ```
     <header class="header">
      <a href="" class="logo">Welcome</a>
      <nav><img src="image/Union.svg" alt="Menu-button" class="hamburger-menu"></nav>
    </header>
 ```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <main>
+      <section class="headline">
+        <h1 class="headline-title">Hey there. I'm Alex. <br/><span class="headline-2">I'm a software developer.</span> </h1>
+        <p class="para"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.</p>
+        <i class="fab fa-github"></i>
+        <i class="fab fa-linkedin-in"></i>
+        <i class="far fa-hand-peace"></i>
+        <i class="fas fa-dove"></i>
+        <i class="fab fa-medium-m"></i>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] Following project requiremnets, kindly wrap your icons in `ul` element with each icons inside `a` tag, this makes your social icons clickable and can navigate to the particular link when clicked. your code should look like this
```
<ul>
<li><a href=""><i class="fab fa-github"></i></a></li>
<li><a href=""> <i class="fab fa-linkedin-in"></i></a></li>
<li><a href=""><i class="far fa-hand-peace"></i></a></li>
<li><a href=""><i class="fas fa-dove"></i></a></li>
<li><a href=""><i class="fab fa-medium-m"></i></a></li>
</ul>

File: index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header class="header">
```
Comment (VICTOR-CHUKWU) [inline index.html:12]: - [ ] t is a good practice for a frontend developer to follow the Figma design given. Though your design is not bad, I think it is better to follow the given mobile menu design.
  your design 👇 
         
image
 Figma design 👇 
      
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @NatiG25  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      <a href="" class="logo">Welcome</a>
```
Comment (Amrhub) [inline index.html:13]: - [ ] when using `<a>` anchor tag by default it has `text-decoration: underline` as CSS property we need to disable that to make sure we match Figma design

File: index.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        <h1 class="headline-title">Hey there. I'm Alex. <br/><span class="headline-2">I'm a software developer.</span> </h1>
+        <p class="para"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.</p>
+        <ul>
+        <li><a href=""><i class="fab fa-github"></i></a></li>
+        <li><a href=""><i class="fab fa-linkedin-in"></i></a></li>
+        <li><a href=""><i class="far fa-hand-peace"></i></a></li>
+        <li><a href=""><i class="fas fa-dove"></i></a></li>
+        <li><a href=""><i class="fab fa-medium-m"></i></a></li>
+        </ul>
```
Comment (Amrhub) [inline index.html]: - [ ] when using `ul` we need to make sure we understand it has default CSS properties as well
1- it list item vertically 
2- it adds a bullet listing using the `list-style` property in CSS 
let's make sure we follow the design

Yours | Figma design
:--:|:--:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @NatiG25, 👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @NatiG25 , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**